services/monitoring/docker-compose.yml.j2

```diff
@@ -288,8 +304,10 @@ services:
       resources:
         limits:
           memory: 512M
+          cpus: "0.5"
```

Comment:
This one as well. I think it will spike if it requires to do some image rendering.

Reply:
I made it 2